src/map/packets/change_music.cpp

```diff
@@ -35,6 +35,6 @@ CChangeMusicPacket::CChangeMusicPacket(uint8 BlockID, uint8 MusicTrackID)
     this->setType(0x5F);
     this->setSize(0x08);
 
-    ref<uint8>(0x04) = BlockID;      // block
-    ref<uint8>(0x06) = MusicTrackID; // music
+    ref<uint16>(0x04)  = BlockID;      // block
```

Comment:
Just a minor thing, extra space got added here.

Reply:
Done. Thanks for the catch!